api/apps/services/models_api_service.py

```
@@ -200,49 +212,79 @@ def list_tenant_default_models(tenant_id: str):
     :param model_instance: instance name
     :param model_name: model name
     :param model_type: model type (chat, embedding, rerank, asr, vision, tts, ocr)
+    :param model_id: tenant_model id
     :return: (success, result_or_error_message)
     """
     field_name = MODEL_TYPE_TO_FIELD.get(model_type)
     if not field_name:
         return False, f"model type '{model_type}' is invalid"
+    tenant_field_name = MODEL_TYPE_TO_TENANT_ID_FIELD.get(model_type)
 
     e, tenant = TenantService.get_by_id(tenant_id)
```

Ruff (0.15.20)

[warning] Unpacked variable `tenant` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)